SPEC.md

```diff
@@ -9007,9 +8795,10 @@ workflow test_select_all {
   }
 
   Array[Int] fivethree = select_all([maybe_five, maybe_four_but_is_not, maybe_three])
+  Array[Int] expected = [5, 3]
 
   output {
-    Boolean is_true = fivethree == [5, 3]
+    Boolean is_true = length(fivethree) == 2 && fivethree == expected
```

Comment:
I don't think we need the length check here, just checking if it matches `expected` is good enough.